### PR TITLE
added `wipe` make command

### PR DESCRIPTION
**Pull request by domx4q**
As suggested in the README.md file, I've added the wipe command to clear the `local_data` folder.

### CODE REVIEW

**Review by imartinez — CHANGES_REQUESTED** [review]
Code changes LGTM! 

Pending to migrate:
- Document the new command it in docs/description.md (I think the section ### Reset Local documents database would be the one to update)
- After changing the documentation, run `make api-docs` and commit the generated changes - it is the way to publish the changes in the documentation.

**Comment by domx4q** [comment]
I've added the command to the description.md file and ran the `make api-docs` command. Now there are some changes in the openapi.json file where I'm not sure if it should be so.

File: docs/openapi.json
[the remark this answers is not in the document]
Comment (domx4q) [reply docs/openapi.json:1]: I've rebased and there are still this weird changes like in line 394. But I think its just a new formating. But I wasn't sure.

**Review by imartinez — APPROVED** [review]

**Review by lopagela — APPROVED** [review]
Good to go - openapi documentation have been also updated after my changes to get rid of `enum` kwargs passed to extensions of `BaseModel`

Good to go!

**Comment by lopagela** [comment]
I restarts the CICD checks on the commit as there seems to be a glitch preventing the merging of this PR.

**Comment by domx4q** [comment]
> I restarts the CICD checks on the commit as there seems to be a glitch preventing the merging of this PR.

I think it's stuck again. I can't see all you repo settings, maybe this article could help.
https://github.com/orgs/community/discussions/26698

**Comment by lopagela** [comment]
> > I restarts the CICD checks on the commit as there seems to be a glitch preventing the merging of this PR.
> 
> I think it's stuck again. I can't see all you repo settings, maybe this article could help. https://github.com/orgs/community/discussions/26698

Thank you @domx4q ! That was indeed the root cause 😉 

Feel free to come and say hi in the discord of this project, [channel](https://discord.com/channels/1164200432894234644/1167405247140986901) `#contributors`

**Pull request merged**